src/Components/AutoComplete/AutoComplete.tsx

```diff
@@ -0,0 +1,51 @@
+import { Autocomplete, FormControl, FormHelperText, TextField } from '@mui/material';
+import { useMemo } from 'react';
+import { FormattedMessageType } from '../../Types/Questions';
+
+type Props = {
+  value: string | null;
+  setValue: (newValue: string | null) => void;
+  options: string[];
+  label: FormattedMessageType;
+  showError: boolean;
+  errorMessage: FormattedMessageType;
+};
+
+export default function AutoComplete({ options, label, value, setValue, showError, errorMessage }: Props) {
+  const autoCompleteOptions = useMemo(() => {
+    return options.map((value) => {
+      return { label: value };
+    });
+  }, [options]);
+
+  const autoCompleteValue = value !== null ? { label: value } : null;
+
+  return (
+    <FormControl sx={{ mt: 1, mb: 2, minWidth: 210, maxWidth: '100%' }} error={showError}>
+      <Autocomplete
+        selectOnFocus
+        clearOnBlur
+        handleHomeEndKeys
+        isOptionEqualToValue={(option, value) => option.label === value.label}
+        options={autoCompleteOptions}
+        value={autoCompleteValue}
+        onChange={(_, newValue) => {
```

Comment:
What does this empty argument before the comma do?

Reply:
I am assigning the `event` parameter to the `_` variable because I don't use it.